bundles/org.openhab.binding.renault/src/main/java/org/openhab/binding/renault/internal/handler/RenaultHandler.java

```diff
@@ -205,6 +206,28 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                     }
                 }
                 break;
+            case RenaultBindingConstants.CHANNEL_PAUSE_MODE:
+                if (command instanceof RefreshType) {
+                    reschedulePollingJob();
+                } else if (command instanceof OnOffType) {
+                    try {
+                        PauseMode newMode = PauseMode.valueOf(command.toString());
+                        MyRenaultHttpSession httpSession = new MyRenaultHttpSession(this.config, httpClient);
+                        try {
+                            httpSession.initSesssion(car);
+                            httpSession.actionPause(newMode);
+                            car.setPauseMode(newMode);
+                            updateState(CHANNEL_PAUSE_MODE, new StringType(newMode.toString()));
+                        } catch (Exception e) {
+                            logger.warn("Error My Renault Http Session.", e);
+                            Thread.currentThread().interrupt();
```

Comment:
Probably you only want to do this for `InterruptedException`.

Reply:
I make a more clear exception handle on my very last commit
Not sure this is what you would like.

Reply:
You need a separate catch for `InterruptedException`. See how this is handled in the code above.